src/services/webclient.ts

```diff
@@ -407,8 +407,9 @@ export class WebClientService implements threema.WebClientService {
             this.$log.debug('Handover done');
 
             // Ask user for notification permission
-            this.$log.debug('Check notification permission...');
-            this.notificationService.requestNotificationPermission();
+            this.$log.debug('Check notification API and fetch initial settings...');
+            this.notificationService.checkNotificationAPI();
+            this.notificationService.fetchSettings();
```

Comment:
Since these methods should always be called together, could you make them private and add a new public function `notification.Service.init()` that calls those other two methods? Might prevent bugs in the future.